realm/realm-library/src/main/java/io/realm/RealmResults.java

```diff
@@ -138,6 +143,279 @@ public boolean load() {
     }
 
     /**
+     * Sets the value to {@code null} for the given field in all of the objects in the result collection.
+     *
+     * @param fieldName name of the field to update.
+     * @throws IllegalArgumentException if field name doesn't exist or isn't nullable.
+     * @throws RealmException if the field is a {@link io.realm.annotations.PrimaryKey} field.
+     */
+    public void setNull(@Nonnull String fieldName) {
```

Comment:
Since you are handling `null` in the other methods using `checkAndSetNullValue()` it seems a bit odd to have this public. Although there is a symmetry argument with `DynamicRealmObject`, so it is probably fine